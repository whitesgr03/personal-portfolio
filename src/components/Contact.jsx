import Icon from "@mdi/react";
import { mdiEmailOutline } from "@mdi/js";

const Contact = () => {
	return (
		<div className="contact">
			<h2>Contact me</h2>
			<p>
				Please get in touch if you think our work could be mutually
				beneficial!
			</p>
			<address>
				<Icon path={mdiEmailOutline} />
				<p>whitesgr03@gmail.com</p>
			</address>
		</div>
	);
};

export default Contact;
