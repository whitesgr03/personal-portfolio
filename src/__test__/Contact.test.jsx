import { render } from "@testing-library/react";

import Contact from "../components/Contact";

describe("Renders Footer Component", () => {
	it("Should return Footer DOM", () => {
		const { container } = render(<Contact />);

		expect(container).toMatchSnapshot();
	});
});
