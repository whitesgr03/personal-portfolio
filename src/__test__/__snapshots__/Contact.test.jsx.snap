// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Component Should render Contact component 1`] = `
<div>
  <div
    class="contact"
  >
    <h2>
      Contact me
    </h2>
    <p>
      Please get in touch if you think our work could be mutually beneficial!
    </p>
    <address>
      <svg
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M22 6C22 4.9 21.1 4 20 4H4C2.9 4 2 4.9 2 6V18C2 19.1 2.9 20 4 20H20C21.1 20 22 19.1 22 18V6M20 6L12 11L4 6H20M20 18H4V8L12 13L20 8V18Z"
          style="fill: currentColor;"
        />
      </svg>
      <p>
        whitesgr03@gmail.com
      </p>
    </address>
  </div>
</div>
`;
